desktop/bin/build-desktop-config.js

```diff
@@ -22,40 +22,11 @@ try {
 	}
 }
 
-const config = JSON.stringify( Object.assign( base, env ), null, 2 );
-
-fs.writeFileSync( path.join( DESKTOP_DIR, 'config', 'config.json' ), config, 'utf-8' );
-
-/*
- * Verify that the `config/secrets.json` file exists. It contains OAuth keys that the app needs
- * to perform user login
- */
-let secretsString;
-try {
-	secretsString = fs.readFileSync( path.join( DESKTOP_DIR, 'config', 'secrets.json' ), 'utf-8' );
-} catch {
-	console.error( 'Config file config/secrets.json does not exist. Please create one.' );
-	process.exit( 1 );
+if ( process.env.WINDOWS_STORE ) {
```

Comment:
We can set a `WINDOWS_STORE` env var in Circle at build time, which will allow us override whether or not the updater is enabled.